ci/matrix.yaml

```diff
@@ -84,6 +87,7 @@ pull_request:
     - {cuda: *cuda_curr,     os: 'windows2022', cpu: 'amd64', compiler: *msvc2019, std: [14, 17],         jobs: ['build']}
     - {cuda: *cuda_curr,     os: 'windows2022', cpu: 'amd64', compiler: *msvc2022, std: [14, 17, 20],     jobs: ['build']}
     - {cuda: *cuda_curr,     os: 'ubuntu22.04', cpu: 'amd64', compiler: *oneapi,   std: [11, 14, 17],     jobs: ['build']}
+    - {cuda: *cuda_curr,     os: 'ubuntu22.04', cpu: 'amd64', compiler: *nvhpc,    std: [11, 14, 17, 20], jobs: ['build']}
```

Comment:
note for the future: This is a little wonky because it would lead someone to believe that they could control the CUDA version separate from the nvhpc version. The current matrix logic isn't built to expect that a host compiler would come with its own CTK.

Reply:
It is but I would rather not completely rewrite the whole matrix for that one use case